frontend/app-preview/src/views/LandingPage.module.css

```diff
@@ -39,3 +39,20 @@
   background-color: #efefef;
   z-index: 1;
 }
+
+.previewLimitationsInfo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 10px;
+  background-color: #efefef;
```

Comment:
Det er litt skummelt å bruke position:fixed fordi elementene potensielt kan overlappe hverandre. Hva tenker du om å endre Landing Page til å bruke et grid med én kolonne og to rader? Den første raden kan være for forhåndsvisning, mens den andre raden kan brukes til informasjon om forhåndsvisningen. 🤔

Slik det er nå, vil varselboksen flyte over bunnen av skjemaet hvis du skroller helt ned.
![image](https://github.com/Altinn/altinn-studio/assets/46874830/b6a03398-1142-4604-b8f8-2e0caa59e240)


Nå går jeg litt inn i UX-området - men jeg mistenker at brukere leser top-down, så kan være at budskapet om hva forhåndsvisning er blir oppdaget etter at har trykket rundt i skjema. Kunne det vært et alternativ å flytte informasjonen og ha en større forskjell fremstillingen? Slik at det er mindre mulighet for misforståelse om den alert-komponenten er en del av skjema eller ikke? Dette var bare løse tanker. 🤔 😅  

![image](https://github.com/Altinn/altinn-studio/assets/46874830/5d523f56-4428-4a6a-9a43-485482fb8e24)

Reply:
Det er kanskje et behov her for å kunne fjerne denne meldingen, da den hvertfall som en alert tar mye plass og oppmerksomhet i bildet (særlig på "lage siden" der plassen er mindre). Om det er en statisk beskjed som skal stå der, så bør det være en mulighet oyr å skjule, eller X ut meldingen.

En rød melding i toppen kan kanskje forveksles med en feilmelding, og om den får en mer nøytral bakgrunnsfarge kan den bli mer usynlig igjen enn en alert i bånn, eller i toppen av skjema. 
Jeg ser ingen problem med å flytte en alert øverst, men da vil jeg anta at behovet for å kunne skjule denne meldingen er desto større, da den blir mer synlig. Visningen/ bruk av bredden må også vise tydelig at dette ikke er en alert i skjema, men for preview-visningen. 

Et annet alternativ er å ha en hjelpetekst-aktig komponent man kunne trykke på hvor det stod noe om hvordan preview virket. Men da spørs det om det dekker det behovet dere ser. Skal brukeren varsles når de åpner preview, eller holder det at informasjonen er tilgjengelig i samme vindu?

Reply:
Det var informativ @Annikenkbrathen. 🤩 Jeg er veldig enig i at den oransje fargen blir noe ekstrem, men den var bare for å illustrere poenget. Men som du sier så er jeg enig i at en mer nøytral farge så blir den usynlig.

Når det gjelder tooltip, så vil kanskje ikke alle få med seg informasjonen? 🤔 Jeg stemmer for å flytte alert til toppen og tilby mulighet for å lukke den med (x) kryss, samt velge "Skjul for alltid". Så bruke bredden til å vise at dette ikke er en del av skjema. 😄

Reply:
Ja, det høres ut som en god ide. Jeg foreslår at vi prøver det, så kan vi teste med brukere hvordan den funker når det er oppe 🤔